runtime/src/main/java/org/corfudb/protocols/wireprotocol/TxResolutionInfo.java

```diff
@@ -46,6 +50,7 @@ public TxResolutionInfo(UUID txId, long snapshotTimestamp) {
         this.snapshotTimestamp = snapshotTimestamp;
         this.conflictSet = Collections.emptyMap();
         this.writeConflictParams = Collections.emptyMap();
+        this.poisonedStreams = Collections.emptySet();
```

Comment:
isn't it better to make this null (cheaper serialization), and check if it is null on the receiving end? token request performance matters.